cmd/minikube/cmd/config/config.go

```diff
@@ -105,9 +105,14 @@ var settings []Setting = []Setting{
 
 var ConfigCmd = &cobra.Command{
 	Use:   "config SUBCOMMAND [flags]",
-	Short: "Modify minishift config",
-	Long: `config modifies minishift config files using subcommands like "minishift config set vm-driver kvm"
-Configurable fields: ` + "\n\n" + configurableFields(),
+	Short: "Modifies Minishift configuration properties.",
+	//NEEDINFO: can we do this any time? or do we need to stop the cluster first? how many config files can we have?
+	Long: `Modifies Minishift configuration properties. The configuration properties in the file are parallel to
```

Comment:
"parallel" is probably the wrong word here, I want to say that these config properties are equivalent to options in the minishift start command.